code/game/objects/items/weaponry.dm

```diff
@@ -113,6 +113,19 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	throw_range = 5
 	armour_penetration = 35
 
+/obj/item/claymore/carrot
+	name = "carrot sword"
+	desc = "A full-sized carrot sword. What else did you expect from a carrot that big?"
+	icon_state = "carrot_sword"
+	inhand_icon_state = "carrot_sword"
+	worn_icon_state = "carrot_sword"
+	force = 25
+	throwforce = 10
+	throw_speed = 3
+	throw_range = 7
+	armour_penetration = 5
+	block_chance = 10
```

Comment:
25 force is definitely too much

Keep in mind carrots are readily available to prisoners. Maybe 15, 16 is more reasonable. 


Also it will conduct electricity, inheriting that from parent, not sure if you care.